core/schema_utils.py

```diff
@@ -544,11 +544,11 @@ def is_valid_math_expression(obj: str, algebraic: bool) -> bool:
         if not expression_parser.is_valid_expression(obj): # type: ignore[no-untyped-call]
             return False
 
-        expression_is_algebraic = expression_parser.is_algebraic(obj) # type: ignore[no-untyped-call]
-        # If the algebraic flag is true, expression_is_algebraic should
-        # also be true, otherwise both should be false which would imply
-        # that the expression is numeric.
-        return not algebraic ^ expression_is_algebraic
+        expression_contains_variables = (
+            expression_parser.contains_variables(obj)) # type: ignore[no-untyped-call]
+        # This validates the check that ensures that
```

Comment:
Try to use up as much of the 80 chars as possible in each line when writing comments and docstrings.

Also this is unclear as written -- what does 'validates the check" mean? See if you can express it more simply -- you'll want either "validate" or "check" since they mean the same thing.

Reply:
Fixed.